src/contracts/market-place.ts

```diff
@@ -20,6 +21,54 @@ export type MarketResponse = unknown[] & {
 
 export class MarketPlace {
 
+    /**
+     * Euler's `Markets` contract proxy addresses per chain.
+     *
+     * @remarks
+     * In order to retrieve the interest rate (supply APY) of a Euler market,
+     * we need the static proxy address of the `Markets` contract.
+     *
+     * Should this address change in the future, you can set it via this static
+     * property. The index of each entry is the chain id of Euler's Markets
+     * contract deployment, currently mainnet and ropsten have deploys.
+     *
+     * https://docs.euler.finance/protocol/addresses
+     */
+    static EULER_ADDRESSES: Record<number, Record<'MARKETS', string>> = {
```

Comment:
sure. there's prob a few ways to solve this. 

I'm prob down with whatever you land on. you are just as bitchy as me about stuff like this.

no. you are worse.

;)

Reply:
yeah, i figured doing it like this allows you to have a set-and-forget kinda feel to it... 
```ts
import { MarketPlace } from '@swivel-finance/swivel-js';

// this could be moved to a config file which gets imported once at script startup
MarketPlace.EULER_ADDRESSES = {
  1: { MARKETS: '0xmyNewCustomAddress' },
};

// now you don't have to worry about addresses anymore and your code doesn't change
const marketPlace = new MarketPlace(...);
const interestRate = await marketPlace.getInterestRate(...);
```